draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -559,22 +560,23 @@ To this end, Verifiers MUST follow the following steps (or equivalent):
     2. Validate the signature over the SD-JWT.
     3. Validate the Issuer of the SD-JWT and that the signing key belongs to this Issuer.
     4. Check that the SD-JWT is valid using `nbf`, `iat`, and `exp` claims, if provided in the SD-JWT, and not selectively disclosed.
-    5. Check that the `_sd_alg` claim is present and its value is understood and the hash algorithm is deemed secure.
- 4. Create a copy of the SD-JWT payload, if required for further processing.
- 5. Process the Disclosures. For each Disclosure provided:
-    1. Calculate the digest over the base64url string as described in (#hashing_disclosures).
-    2. Find all `_sd` keys in the SD-JWT payload. For each such key:
+    5. Check that the `_sd_alg` claim value is understood and the hash algorithm is deemed secure.
+ 4. Process the Disclosures and `_sd` keys in the SD-JWT as follows:
+    1. Create a copy of the SD-JWT payload, if required for further processing.
+    2. For each Disclosure provided:
+       1. Calculate the digest over the base64url string as described in (#hashing_disclosures).
```

Comment:
```suggestion
    2. For each Disclosure provided:
        1. If the Disclosure is not a JSON-encoded array of three elements, the Verifier MUST reject the Presentation.
        2. Calculate the digest over the base64url-encoded string as described in (#hashing_disclosures).
```
the rest of the document seem to use `base64url-encoded`, we might want to do a clean up PR on that if not in this one

Reply:
base64url-encoded works for me.

Reply:
base64url-encoded works for me too. So it'd be
```suggestion
    2. For each Disclosure provided:
       1. Calculate the digest over the base64url-encoded string as described in (#hashing_disclosures).
```